functions/Get-DbaTraceFlag.ps1

```diff
@@ -0,0 +1,96 @@
+﻿FUNCTION Get-DbaTraceFlag {
+	<#
+		.SYNOPSIS
+			Gets SQL Trace Flags information for each instance(s) of SQL Server.
+
+		.DESCRIPTION
+			The Get-DbaTraceFlag returns connected SMO object for SQL Trace Flags information for each instance(s) of SQL Server.
+
+		.PARAMETER SqlInstance
+			SQL Server name or SMO object representing the SQL Server to connect to. This can be a collection and receive pipeline input to allow the function to be executed against multiple SQL Server instances.
+
+		.PARAMETER SqlCredential
+			SqlCredential object to connect as. If not specified, current Windows login will be used.
+
+		.PARAMETER TraceFlag
+			Use this switch to filter to a specific Trace Flag.
+
+		.PARAMETER Silent
+			Use this switch to disable any kind of verbose messages.
+
+		.NOTES
+			Tags: Trace, Flag
+			Original Author: Kevin Bullen (@sqlpadawan)
+
+			References:  https://msdn.microsoft.com/en-us/library/microsoft.sqlserver.management.smo.server.enumactivecurrentsessiontraceflags(v=sql.120).aspx
```

Comment:
We don't generally provide links to the SMO object utilized, this would be more to note in your PR.

If you would update that URL to this one: `https://docs.microsoft.com/en-us/sql/t-sql/database-console-commands/dbcc-traceon-trace-flags-transact-sql`